tuneful-client/src/Services/token-service.js

```diff
@@ -13,8 +13,8 @@ const TokenService = {
   hasAuthToken() {
     return !!TokenService.getAuthToken()
   },
-  makeBasicAuthToken(userName, password) {
-    return window.btoa(`${userName}:${password}`)
+  makeBasicAuthToken(email, password) {
+    return window.btoa(`${email}:${password}`)
```

Comment:
are you using this or token auth at the moment?

Reply:
Just reviewed this. It's just a services file. The method exists from before when I was using basicauth. It is not used in the code anymore